### ISSUE TITLE
lib_xs3_math.cmake doesn't work with SDK

### ISSUE THREAD

**Issue by astewart-xmos** [xmos/lib_xs3_math#83]
The xcore SDK includes lib_xs3_math.cmake to get the lib_xs3_math sources and includes, but the version currently in develop of lib_xs3_math doesn't work.

First, it uses relative paths that don't work when it's being `include`d.

Second, apparently the CMAKE_SYSTEM_NAME when building with the SDK is `XMOS` and not `XCORE`, so the assembly sources don't make it into `LIB_XS3_MATH_SOURCES`.

**Comment by astewart-xmos** [comment]
Apparently GitHub won't let me attach a .cmake file to an issue.  BECAUSE REASONS.

So I add the extension .txt.  DID THAT MAKE THE WORLD SAFER GITHUB?? DID IT!??

[lib_xs3_math.cmake.txt](https://github.com/xmos/lib_xs3_math/files/7730243/lib_xs3_math.cmake.txt)

I would have just committed a branch with the fix, but apparently the submodule is  a shallow clone and shallow updates aren't allowed.

**Comment by keithm-xmos** [comment]
This bug may no longer be relevant once https://github.com/xmos/xcore_sdk/issues/290 is implemented.

**Comment by astewart-xmos** [comment]
This should be good once PR xmos/xcore_sdk#386 is merged into xcore_sdk
